src/ci/resources/papi_v2_34_application.conf

```diff
@@ -39,3 +39,15 @@ backend.providers.Papi-Caching-No-Copy.config.filesystems.http {
   project = "broad-dsde-cromwell-dev"
   auth = "service_account"
 }
+backend.providers.Papiv2USADockerhub.config.filesystems.http {
+  project = "broad-dsde-cromwell-dev"
+  auth = "service_account"
+}
+
+# The default v2 config sets this to "user_service_account" but that value isn't handled correctly in
+# versions prior to 37 so override this to be `service_account`.
+backend.providers.Papi.config.dockerhub.auth = "service_account"
+backend.providers.Papiv2.config.dockerhub.auth = "service_account"
+backend.providers.Papiv2RequesterPays.config.dockerhub.auth = "service_account"
+backend.providers.Papi-Caching-No-Copy.config.dockerhub.auth = "service_account"
+backend.providers.Papiv2USADockerhub.config.dockerhub.auth = "service_account"
```

Comment:
Did you use this file manually?

ToL / Conflicting thoughts:

- This orphaned conf can be deleted. It's no longer automatically used by the CI tests in `develop`.

but

- There are still GOTC folks using 34, so one may need to prove to them that cromwell can do a multi-version jump.

Reply:
I'm fine either way. Under the "they who last touched it" principle be prepared to answer questions about this file though. 😉